src/shared/Core/Settings.cs

```diff
@@ -536,7 +540,11 @@ public bool IsInteractionAllowed
             }
         }
 
-        public bool GetTracingEnabled(out string value) => _environment.Variables.TryGetValue(KnownEnvars.GcmTrace, out value) && !value.IsFalsey();
+        public bool GetTracingEnabled(out string value) =>
+            TryGetSetting(KnownEnvars.GcmTrace,
+                KnownGitCfg.Credential.SectionName,
+                KnownGitCfg.Credential.Trace,
+                out value) && !value.IsFalsey();
```

Comment:
Maybe make this `IsTruthy()` to align with the `IsDebuggingEnabled` check above and the `IsSecretTracingEnabled`/`IsMsalTracingEnabled` checks below.

Reply:
We need to keep this to `!IsFalsey` because we can set this variable to something like a file path which is considered "enabled".

Value|`TryGetSetting`|`IsTruthy`|`IsFalsey`|`TryGetSetting && !IsFalsey`
-|-|-|-|-
`null`|❌|❌|❌|❌
`"/tmp/file.txt"`|✅|❌|❌|✅
`true`/`1`/`"yes"`/`"on"`|✅|✅|❌|✅
`false`/`0`/`"no"`/`"off"`|✅|❌|✅|❌

Reply:
Lovely truth table - explains very nicely, thank you!